libraries/exoplayer/src/main/java/androidx/media3/exoplayer/source/ads/ServerSideAdInsertionMediaSource.java

```diff
@@ -427,20 +427,22 @@ public void onLoadStarted(
       int windowIndex,
       @Nullable MediaPeriodId mediaPeriodId,
       LoadEventInfo loadEventInfo,
-      MediaLoadData mediaLoadData) {
+      MediaLoadData mediaLoadData,
+      int retryCount) {
     @Nullable
     MediaPeriodImpl mediaPeriod =
         getMediaPeriodForEvent(mediaPeriodId, mediaLoadData, /* useLoadingPeriod= */ true);
     if (mediaPeriod == null) {
-      mediaSourceEventDispatcherWithoutId.loadStarted(loadEventInfo, mediaLoadData);
+      mediaSourceEventDispatcherWithoutId.loadStarted(loadEventInfo, mediaLoadData, retryCount);
```

Comment:
this call (and the one below in this method) are basically forwarding from an inner `MediaSource` instance. SImilar to `CompositeMediaSource.ForwardingEventListener` we should implement both the old and new methods here, and forward each one to the respective `EventDispatcher` method.

Otherwise a custom `MediaSource` that hasn't yet switched to dispatching the new method will have all its `onLoadStarted` events dropped at this point.